test/blocks/merch/merch.test.js

```diff
@@ -154,15 +154,39 @@ describe('Merch Block', () => {
   });
 
   describe('CTAs', () => {
-    it('merch link to CTA', async () => {
+    it('merch link to CTA, default values', async () => {
       const el = document.querySelector('.merch.cta');
       const { nodeName, textContent, dataset } = await merch(el);
       expect(nodeName).to.equal('A');
       expect(textContent).to.equal('Buy Now');
       expect(dataset.template).to.equal('checkoutUrl');
       expect(dataset.promotionCode).to.equal(undefined);
+      expect(dataset.checkoutWorkflow).to.equal(undefined);
+      expect(dataset.checkoutWorkflowStep).to.equal('email');
+      expect(dataset.checkoutClientId).to.equal('adobe_com');
+      expect(dataset.checkoutMarketSegment).to.equal(undefined);
+    });
+
+    it('merch link to CTA, config values', async () => {
+      setConfig({
```

Comment:
I believe merch block should have its own config system.

Reply:
AFAIK `setConfig` is consumer level configuration and for merch, we may need contextual parameters.
Actually, we should start with metadata and see how it goes.
since each consumer has own metadata, setting the client_id and checkout workflow (only ucv2 ones as ucv3 by default) in metadata.xls is probably the simplest approach.

Reply:
the best part is that, if we will need a checkout type in metadata - we can add it at any moment without significant cost.

Reply:
@3ch023 you can specify where metadata appears. Either on a page by page basis or in all pages in a folder etc.